src/ltablib.js

```diff
@@ -4,7 +4,10 @@ const assert  = require('assert');
 
 const lua     = require('./lua.js');
 const lauxlib = require('./lauxlib.js');
-const llimit  = require('./llimit.js');
+const luaconf = require('./luaconf.js');
+
+const INT_MAX = 2147483647;
+const INT_MIN = -2147483648;
```

Comment:
INT_MIN doesn't seem to get used anywhere? Shouldn't the linter yell at us for this?